my-app/src/app/toolbar/toolbar.component.css

```diff
@@ -0,0 +1,87 @@
+* {
+    height: 100%;
+    margin: 0;
+}
+
+.nav {
+    border-bottom: 1px solid #EAEAEB;
+    text-align: right;
+    height: 84px;
+    background: rgb(4, 116, 192);
+    background: linear-gradient(90deg, rgba(4, 116, 192, 1) 32%, rgba(17, 118, 196, 1) 43%, rgba(236, 192, 39, 1) 66%);
+    display: flex;
+    flex-direction: row;
+}
+
+.logo {
+    width: 19rem;
+    height: 8rem;
+    margin-right: 42rem; 
+    background-image: url("../images/logo.png");
```

Comment:
Typically the best practice for images in an angular app is to put them in assets/img folder.